README.md

```diff
@@ -93,6 +93,12 @@ The default recipe includes the appropriate recipe based on the node's
 This recipe ensures the platform-specific configuration directory
 (`/etc/chef`) is created, and renders the configuration file
 (`push-jobs-client.rb`) using the `whitelist` attribute.
+Any environment variables can be set using `environment_variables`
+attribute with key value pairs.
+If you need to further customize the push-jobs-client.rb file,
+you can provide your own template in a wrapper cookbook and set the
```

Comment:
If user decides to use their own template in order for it to work we they will need to set this attribute otherwise it will not work and it might get them sometime to find why and look through code.